docs/getting-started-guides/azure.md

```diff
@@ -0,0 +1,104 @@
+---
+---
+
+* TOC
+{:toc}
+
+## Prerequisites
+
+1. An Azure Subscription. If you don't already have one, you may create one on [azure.microsoft.com](https://azure.microsoft.com).
```

Comment:
I'd thought about it, but the link in there now is less likely to change and has a huge green Free button in the middle of it.